### DISCUSSION TITLE
Stepping back a dependency

### DISCUSSION THREAD

**Discussion by fretwellian** [wire-elements/spotlight#90] [Q&A]
Just checking that I am not missing anything obvious before I try and implement something. 

Is there an easy way to step back a dependency once you've made a selection. For instance being able to hit delete on an empty search field to take you back to a team selection if you choose the wrong one. Currently it's `escape` then `cmd` +`k` to start again.

**Comment by PhiloNL** [comment]
Hi @fretwellian,
At the moment, there is no way to go back except, as you mentioned, by closing and opening the Spotlight.
I am working on a Pro version that does support this (and a lot of additional features). 
https://twitter.com/Philo01/status/1556618729860972546
